templatetags/htk_tags.py

```diff
@@ -458,3 +458,18 @@ def credit_card_icon(credit_card_brand):
     else:
         credit_card_icon = DEFAULT_CREDIT_CARD_ICON
     return credit_card_icon
+
+
+@register.simple_tag()
+@mark_safe
+def print_js_routes():
+    from htk.utils.js_route_serializer import build_routes
+    js_obj = htk_setting('HTK_JS_ROUTES_JS_OBJECT_NAME')
+    urls = build_routes(as_json=True)
+    html = '<script type="text/javascript">'
+    html += '{js_obj} = Object.assign({js_obj} || {{}}, {{ urls: {urls} }});'.format(
```

Comment:
- Personally, I feel that the `Object.assign(window.Site)` code is going to be very specific.
- Note that `urls` as a key is hardcoded
- I think the `<script type="text/javascript">` is also a bit overextending in reach
- I think the most useful is `build_routes(as_json=True)`
- With these things simplified, would this tag still need `@mark_safe` ?